PandaGrom/hometask-5/4.8.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM film cascade WHERE title = 'The Gentlemen'
```

Comment:
I don't think you should specify `cascade` option while deleting
Please, doublecheck in the documentation